environment-dev.yml

```diff
@@ -10,27 +10,25 @@ dependencies:
   - haversine >=2.8.0
   - importlib-metadata <8.0  # Required for xesmf >=8.4.0,<8.6 to work on Windows
   - leafmap
-  - numpy >=1.23,<2.0.0
-  - pandas >=2.2.0
```

Comment:
Suggestion: Keep the lower pins in, since it reduces the solver time and is based closely on SciPy SPEC-0 recommendations.